editor/src/main/res/values/attrs.xml

```diff
@@ -24,18 +23,25 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 
 <resources>
+    <!-- Style attribute for CodeEditor -->
+    <attr name="codeEditorStyle" format="reference" />
 
-    <declare-styleable name="editor">
-        <attr name="text" format="string"/>
-        <attr name="textSize" format="dimension"/>
-        <attr name="dividerWidth" format="dimension"/>
-        <attr name="lineNumberVisible" format="boolean"/>
-        <attr name="autoCompleteEnabled" format="boolean"/>
-        <attr name="symbolCompletionEnabled" format="boolean"/>
-        <attr name="cursorBlinkPeriod" format="integer"/>
-        <attr name="scrollbarsEnabled" format="boolean"/>
-        <attr name="verticalScrollbarEnabled" format="boolean"/>
-        <attr name="horizontalScrollbarEnabled" format="boolean"/>
+    <declare-styleable name="CodeEditor">
+        <attr name="android:scrollbarTrackHorizontal" />
+        <attr name="android:scrollbarThumbHorizontal" />
+        <attr name="android:scrollbarTrackVertical" />
+        <attr name="android:scrollbarThumbVertical" />
+
+        <attr name="text" format="string" />
+        <attr name="textSize" format="dimension" />
+        <attr name="dividerWidth" format="dimension" />
+        <attr name="lineNumberVisible" format="boolean" />
+        <attr name="autoCompleteEnabled" format="boolean" />
+        <attr name="symbolCompletionEnabled" format="boolean" />
+        <attr name="cursorBlinkPeriod" format="integer" />
+        <attr name="scrollbarsEnabled" format="boolean" />
+        <attr name="verticalScrollbarEnabled" format="boolean" />
+        <attr name="horizontalScrollbarEnabled" format="boolean" />
```

Comment:
A bit off-topic but it appears that you've set up some styleables but never implemented them. If you do implement them, I suggest replacing `text` with `android:text` and `textSize` with `android:textSize` for a better consistency.

Reply:
Ok I'll change when I have time.
I forgot to implement them.